src/settings/FinePolicy/components/EditSections/ReminderFeesSection/ReminderFeesSection.js

```diff
@@ -52,6 +52,28 @@ const ReminderFeesSection = ({ sectionOpen, noticeTemplates, blockTemplates }) =
             />
           </Col>
         </Row>
+        <Row>
+          <Col xs={3}>
+            <Field
+              data-testid="ignoreGracePeriodHoldsTestId"
+              label={<FormattedMessage id="ui-circulation.settings.finePolicy.reminderFees.ignoreGracePeriodHolds" />}
+              name="reminderFeesPolicy.ignoreGracePeriodHolds"
+              dataOptions={dataOptions}
```

Comment:
As a tiny tiny matter of style, I prefer to say `component={Select}` _before_ `dataOptions={dataOptions}`, as the data-options are a refinement of how the selector works. Feel free to ignore this if you don't agree, or if consistent style in this code is to use the order you went with here.

Reply:
good idea @MikeTaylor!